pkg/cvo/sync_worker.go

```diff
@@ -183,15 +183,15 @@ type SyncWorker struct {
 	// of the form exclude.release.openshift.io/<identifier>=true
 	exclude string
 
-	// includeTechPreview is set to true when the CVO should create resources with the `release.openshift.io/feature-gate=TechPreviewNoUpgrade`
-	includeTechPreview bool
+	// requiredFeatureSet is set to true when the CVO should create resources with the `release.openshift.io/feature-gate=TechPreviewNoUpgrade`
```

Comment:
nit: Replace `to true`, etc. with whatever the `string` semantics are supposed to be.